help/saml-authentication.md

```diff
@@ -213,6 +213,48 @@ providers.
 
     Once SAML has been configured, consider also [configuring SCIM](/help/scim).
 
+## Synchronizing group membership with SAML
+
+Zulip includes beta support for syncing group memberships upon user
+login. This allows your users' Zulip groups to be updated based on their
+groups in your IdP's directory.
```

Comment:
we could (maybe should) explain the semantics exactly here - but i also want to keep this brief and not overwhelm with details? We detail this properly in RTD https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#synchronizing-group-membership-with-saml, where it feels more natural

Reply:
Yeah that seems like a fine level of detail. We should link to the ReadTheDocs page though.